Kickstarter-iOS/DataSources/DashboardDataSource.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import Library
+import UIKit
```

Comment:
After moving things around these sorts of `UIKit` and `Foundation` imports became necessary as they are no longer 'implicitly' imported.